src/editor/EditorControls.scss

```diff
@@ -1,11 +1,13 @@
+@import "src/slugline.scss";
+
 .editor-controls {
-  padding: 5px;
+  padding: map-get($map: $spacers, $key: 1);
```

Comment:
Thought: in the future, might be worth defining a helper function for getting spacing.

Reply:
I'll get that in before we merge.